docs: update readme and CS website (#14)
CS-SI/eodag-sentinelsat: README.rst

=================

This is a repository for sentinelsat plugin to `EODAG <https://github.com/CS-SI/eodag>`_.

> It's an `Api <https://eodag.readthedocs.io/en/master/plugins.html>`_ plugin that enables to

Reviewer: ```suggestion
It's an `Api <https://eodag.readthedocs.io/en/latest/plugins.html>`_ plugin that enables to
```

search and download EO products from catalogs implementing the
`SciHub / Copernicus Open Access Hub interface <https://scihub.copernicus.eu/userguide/WebHome>`_.
It is basically a wrapper around `sentinelsat <https://sentinelsat.readthedocs.io>`_, enabling it to be used on eodag.

> .. image:: https://eodag.readthedocs.io/en/master/_static/eodag_bycs.png

Reviewer: ```suggestion
.. image:: https://eodag.readthedocs.io/en/latest/_static/eodag_bycs.png
```

2. Follow the guidelines provided in `EODAG's documentation <https://eodag.readthedocs.io/en/latest/getting_started_guide/configure.html>`_
   to configure the plugin. You can create a dedicated configuration file or edit the default one ``~/.config/eodag/eodag.yml``:

   .. code-block:: yaml

      scihub:
          priority: 2  # Must be higher than the other providers' priorities
          api:
              credentials:
                  username: "PLEASE_CHANGE_ME"  # Your own username
                  password: "PLEASE_CHANGE_ME"  # Your own password

Examples
========

Python API:

.. code-block:: python

   import eodag

   dag = EODataAccessGateway()

   search_results, _ = dag.search(
       productType="S2_MSI_L1C",
       start="2021-03-01",
       end="2021-03-31",
       geom={"lonmin": 1, "latmin": 43, "lonmax": 2, "latmax": 44}
   )
   product_paths = dag.download_all(search_results)

CLI:

.. code-block:: bash

   eodag search \
      --productType S2_MSI_L1C \
      --start 2018-01-01 \
      --end 2018-01-31 \
      --box 1 43 2 44 \
      --storage my_search.geojson

>    eodag download --conf my_config.yml --search-results my_search.geojson

Reviewer: ```suggestion
   eodag download --search-results my_search.geojson
```

    python -m pip install eodag-sentinelsat

Configuration
=============

1. Register to `Scihub <https://scihub.copernicus.eu/userguide/SelfRegistration>`_ to get the required credentials (username/password).

> 2. Follow the guidelines provided in `EODAG's documentation <https://eodag.readthedocs.io/en/master/getting_started_guide/configure.html>`_

Reviewer: ```suggestion
2. Follow the guidelines provided in `EODAG's documentation <https://eodag.readthedocs.io/en/latest/getting_started_guide/configure.html>`_
```

>    to configure the plugin. For instance, create a dedicated configuration file ``my_config.yml``:

Reviewer: ```suggestion
   to configure the plugin. You can create a dedicated configuration file or edit the default one ``~/.config/eodag/eodag.yml``:
```

   .. code-block:: yaml

      scihub:

>           priority: 2  # Must be higher than the other provider's priority

Reviewer: ```suggestion
          priority: 2  # Must be higher than the other providers' priorities
```

          api:
              credentials:
                  username: "PLEASE_CHANGE_ME"  # Your own username
                  password: "PLEASE_CHANGE_ME"  # Your own password

Examples
========

Python API:

.. code-block:: python

   import eodag

>    dag = EODataAccessGateway("my_config.yml")

Reviewer: ```suggestion
   dag = EODataAccessGateway()
```

   search_results, _ = dag.search(
       productType="S2_MSI_L1C",
       start="2021-03-01",
       end="2021-03-31",
       geom={"lonmin": 1, "latmin": 43, "lonmax": 2, "latmax": 44}
   )
   product_paths = dag.download_all(search_results)

CLI:

.. code-block:: bash

   eodag search \

>       --conf my_config.yml \

Reviewer: ```suggestion
```

    pre-commit install
    tox

License
=======

eodag-sentinelsat is licensed under GPLv3.

> See LICENSE file for details.

Reviewer: ```suggestion
See `LICENSE <https://github.com/CS-SI/eodag-sentinelsat/blob/develop/LICENSE>`_ for details.
```
